delegation/index.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: pattern
+title: Delegation
+folder: delegation
+permalink: /patterns/delegation/
+categories: Behavioral
+tags:
+ - Java
+ - Difficulty-Beginner
+---
+
+**Also known as:** Proxy Chains
```

Comment:
They are very similar https://en.wikipedia.org/wiki/Proxy_pattern might be better to rename the aka to Proxy Pattern